### ISSUE TITLE
Automatic track change notification

### ISSUE THREAD

**Issue by MichalSuchy** [Surveyor123/freeradio#2]
Hello. If possible, can you add the option to make Track Changes read by Sapi5 as well? And there might be the possibility of changing whether the track changes to read NVDA or Sapi5. This could be useful in the case when we are actively working with NVDA and the voice does not read the track change. Thanks, Michal.

**Comment by Surveyor123** [comment]
it will be in v2026.19.6
